backend/.kubernetes/kubernetes.yaml

```diff
@@ -0,0 +1,175 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: spellbook
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: spellbook
+  name: spellbook-api
+  labels:
+    app: spellbook-api
+spec:
+  selector:
+    matchLabels:
+      app: spellbook-api
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: spellbook-api
+    spec:
+      containers:
+        - name: spellbook-api-app
+          image: 083767677168.dkr.ecr.us-east-2.amazonaws.com/spellbook-prod-ecr
```

Comment:
Shouldn't we hide this link?
Alternatively, we could directly publish an official Docker Hub image as this project is open source, what do you think?

Reply:
It's a private repository so hiding the link would only add obscurity, the real security is that it requires auth. I could certainly be convinced to consider making a public image, but it would require a decent bit of work to extract the configuration out in a way that would be useful to anyone else. Right now the image kind of relies on the infrastructure looking exactly like ours.

Reply:
Yeah, we'll think about this in the future.